components/PlayerDetailsList/PlayerDetailsList.tsx

```diff
@@ -14,9 +14,9 @@ import styles from './PlayerDetailsList.module.css'
 
 interface Props {
     playerUUID: string
+    playerName: string
     loadingDataFunction: Function
     type: 'auctions' | 'bids'
-    auctions?: Auction[]
```

Comment:
please re-add the auctions here.
We want to have the auctions loaded serverside.

- We render the page server side => The auctions are already rendered, which is good for multiple SEO points
- We try to get as much initial data as possible from the server as the entire pre-rendered page is then cached for all users

Reply:
Ah, understood. In that case, would it be a better idea to also add a `bids` field down the line as well? So that we can render both the initial 12 bids and auctions on the server side

Reply:
Possible, but not necessary, as the initially rendered page is always the auction page.
So the bids dont get rendered, even if they are available. => Bids are always loaded, even when not needed.

Reply:
I might be wrong but I feel that it would be possible that the initially rendered page is bids. This would happen when searching for player X while on player Y's bid's section. This occurs bc prevDetailType is preserved accross player searches.